auth.go

```diff
@@ -516,7 +549,44 @@ func verifyGroupIDMathch(linkID *big.Int, groupID int, requestID uint32, groupID
 	return nil
 }
 
+// VerifyToken performs verification of jws/jwz token using the registered packers in package manager
+func (v *Verifier) VerifyToken(
+	token string,
+	opts ...pubsignals.VerifyOpt,
+) (*protocol.AuthorizationResponseMessage, error) {
+
+	var unpackOpts []iden3comm.PackerParams
+	cfg := pubsignals.VerifyConfig{}
+	for _, o := range opts {
+		o(&cfg)
+	}
+	if cfg.AcceptedProofGenerationDelay > 0 {
+		authV2Set, err := v.verificationKeyLoader.Load(circuits.AuthV2CircuitID)
```

Comment:
I could move this to constructor and be called once. Or cached. I've left it here for readibility